doc/user/content/overview/materialize-cdc.md

```diff
@@ -0,0 +1,237 @@
+---
+title: "Materialize CDC"
+description: "The Materialize CDC format is a format for change datafeeds that has been designed especially to prevent record duplication or mistaken missequencing."
+menu:
+  main:
+    parent: "overview"
+    weight: 4
+---
+
+Change data capture (CDC) tools provide feeds that record any changes to a database. Typically, the feeds are then saved to another platform, like Kafka, for storage or processing. However, sometimes the stream can have missing or duplicate records, or records can be received out of order. For example, if a CDC tool crashes while writing a record, it may retry and write the record again, resulting in a duplicate entry.
+
+The **Materialize CDC format** has been designed to provide a downstream data consumer (like Materialize) with enough information to recognize when records are duplicated or out of order. For a technical deep dive on the subject, see our blog post on [Change Data Capture](https://materialize.com/change-data-capture-part-1/).
+
+Currently, the Materialize CDC format is only supported for [Avro-formatted Kafka sources](/sql/create-source/avro-kafka). If you're interested in using it for another source type, let us know in our [Slack workspace](https://materialize.com/s/chat).
+
+To use the Materialize CDC format, you must:
+
+1. Set up your CDC tool to emit its changefeed in the Materialize CDC format
+2. Define the Materialize CDC format in an Avro schema when you [create a source](../../sql/create-source/avro-kafka/) in Materialize
+
+## Materialize CDC JSON
+
+You use a JSON document to define the format of the record updates transmitted in the changefeed and the format of the separate progress messages which tell the downstream consumer (Materialize) how many updates to expect for a particular logical timestamp. The downstream consumer will then compare the updates it has received to the expected updates and discard duplicates if there are too many.
+
+### Update JSON array
```

Comment:
Here and below with "Progress", I don't know what a "JSON array" is.

Reply:
I combined the separate update and progress examples into a single combined Materialize CDC Avro schema example and removed mention of JSON. Does that work better?